client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -64,14 +77,31 @@ export const EquipmentDetailView = () => {
 
   useEffect(() => {
     if (!data) return;
-    setDraft({ ...data });
+    setDraft({
+      ...data,
+      locationIds: draft?.locationIds
+        ? draft.locationIds
+        : data.locations.nodes.map(location => location.id),
+    });
   }, [data, setDraft]);
 
-  if (isLoading) return <DetailFormSkeleton />;
+  useEffect(() => {
```

Comment:
I'm curious - why not just use `useQuery` rather than `useMutation`? 
that way you don't have to use the effect and call this explicitly